examples/tof-viewer/src/ADIMainWindow.cpp

```diff
@@ -404,7 +404,10 @@ void ADIMainWindow::render() {
         ImGui_ImplOpenGL3_NewFrame();
         ImGui_ImplGlfw_NewFrame();
         ImGui::NewFrame();
-        handleInterruptCallback();
+        if (!m_callbackInitialized) {
+            handleInterruptCallback();
+            m_callbackInitialized = true;
+        }
```

Comment:
Why the need for these changes? The commit message says "Added offline condition in handleInterruptCallback". I can't find any offline condition within these changes.

Reply:
Indeed there is no offline changes in this commit, I will make a separate pr for these changes.

Reply:
Changes removed